modules/openapi-generator/src/main/resources/typescript-nestjs/configuration.mustache

```diff
@@ -1,4 +1,9 @@
+{{#useAxiosHttpModule}}
```

Comment:
`@nestjs/axios` is only in the `package.json` when `useAxiosHttpModule` is truthy. If not, the package is not a dependency, which then fails in the type checks, as it can't be resolved. Other code uses the `HttpService` from nestjs/common in that case, so we do that here as well now, too.